x/gravity/module.go

```diff
@@ -0,0 +1,189 @@
+package gravity
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+
+	"github.com/gorilla/mux"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/spf13/cobra"
+	"github.com/umee-network/umee/x/gravity/simulation"
+
+	"github.com/Gravity-Bridge/Gravity-Bridge/module/x/gravity/client/cli"
+	"github.com/Gravity-Bridge/Gravity-Bridge/module/x/gravity/keeper"
+	"github.com/Gravity-Bridge/Gravity-Bridge/module/x/gravity/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	distrtypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+// type check to ensure the interface is properly implemented
+var (
+	_ module.AppModuleSimulation = AppModule{}
+)
+
+// AppModuleBasic object for module implementation
+type AppModuleBasic struct{}
```

Comment:
We shouldn't have to redefine`AppModuleBasic`. We can just embed the original one defined by the gravity module and override and method(s) that we need to.